CHANGELOG.md

```diff
@@ -6,103 +6,130 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 ## v1224
 
 ### Support SPA, XHR, and session trace features on Chrome for iOS.
```

Comment:
```suggestion
### Support SPA, XHR, and session trace features on Chrome for iOS
```

Reply:
I don't want to make changes outside of what prettier is doing automatically in this PR.